src/main/java/com/benko/exercise/sort/BubbleSorter.java

```diff
@@ -0,0 +1,34 @@
+package com.benko.exercise.sort;
+
+public class BubbleSorter {
+
+    public void sort(int[] numbers) {
+        for (int i = 0; i < numbers.length; i++) {
+            swapElements(numbers);
+        }
+        printNumbers(numbers);
```

Comment:
Sorter was not intended to do printing. That was the main method's job.

Reply:
Done. Now I got it how it works in main method!